### ISSUE TITLE
[Roadmap] Build Process

### ISSUE THREAD

**Issue by RyanTheAllmighty** [ATLauncher/ATLauncher-NEXT#13]
Currently in the initial-code branch we're using CircleCI to do CI and build the application.

This unfortunately doesn't build exactly as we'd need.

We can only do OSX signing from OSX for instance so we cannot sign the resulting artifact for OSX and we also aren't able to use Squirrel for example for updating.

One solution is to use electron-builder and a mix of TravisCI and Appveyor to build natively on all platforms.

Some documentation on it is at https://github.com/electron-userland/electron-builder/wiki/Multi-Platform-Build

**Comment by jamierocks** [comment]
Yay for Travis :D

**Comment by RyanTheAllmighty** [comment]
Yeah unfortunately it seems to not be possible to build for all platforms on a single OS type if you want things such as installers and auto updaters using Squirrel
